tests/test_orders_mapper.py

```diff
@@ -210,3 +276,211 @@ def test_composite_order_with_one_pol_mapping(mapper):
         == "ae1daef2-ddea-4d87-a434-3aa98ed3e687"
     )
     assert composite_order_with_pol["compositePoLines"][0]["cost"]["currency"] == "USD"
+
+
+def test_one_order_one_pol_multiple_notes(mapper):
+    mapper.composite_order_map = {
+        "data": [
+            {
+                "folio_field": "legacyIdentifier",
+                "legacy_field": "order_number",
+                "value": "",
+                "description": "",
+            },
+            {
+                "folio_field": "poNumber",
+                "legacy_field": "order_number",
+                "value": "",
+                "description": "",
+            },
+            {"folio_field": "vendor", "legacy_field": "vendor", "value": "", "description": ""},
```

Comment:
Not that it matters, really, but what is going on with the formatting here?

Reply:
I think it's something automatic, if an object is "smaller" it ends up on one row. It looks weird but I haven't tried to change the setting.

Reply:
Is it Black defaults? Not sure what can be done about that.